src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const chats = props.entries.map((entry, i) => {
+    return (
+      <ChatEntry
```

Comment:
So you see how setting all of those props took up a lot of lines of code? We could make that more concise with this here: 
```javascript
 < ChatEntry toggleLike={props.updateLike} key={entries.id} entry={entry} />
```
Then in each `chatEntry` we could access any prop we needed with `props.entry.myProp`